astropy/modeling/tests/test_models.py

```diff
@@ -516,3 +520,55 @@ def test_voigt_model():
 def test_model_instance_repr():
     m = models.Gaussian1D(1, 2, 3)
     assert repr(m) == '<Gaussian1D(amplitude=1.0, mean=2.0, stddev=3.0)>'
+
+
+@pytest.mark.skipif("not HAS_SCIPY")
+def test_tabular_interp_1d():
+    points = np.arange(0, 5)
+    values = [1., 10, 2, 45, -3]
+    LookupTable = models.tabular_model(1)
```

Comment:
They are equivalent. This actually exercises the code, so it doesn't really matter.